src/components/CountdownTimer/index.tsx

```diff
@@ -0,0 +1,51 @@
+"use client";
+
+import { useCountdownTimer } from "./useCountdownTimer";
+
+export default function CountdownTimer() {
+  const { timeLeft } = useCountdownTimer();
+
+  return (
+    <div className="w-full max-w-full md:max-w-2xl">
+      <div className="backdrop-blur-sm bg-white/85 rounded-xl py-4 md:py-8 shadow-lg">
+        <div className="flex gap-1 md:gap-4 items-end justify-center text-blue-600">
+          <div className="text-center min-w-[3rem]">
+            <div className="text-xs md:text-sm font-bold mb-1 md:mb-3">
+              Days
+            </div>
+            <div className="text-2xl md:text-5xl font-bold tabular-nums">
```

Comment:
こちらの時間を表示する部分もtime要素を使用してもいいかなと思いました。
```suggestion
            <time dateTime={`PT${timeLeft.days}H`} className="block text-2xl md:text-5xl font-bold tabular-nums">
```

Reply:
timeタグ初めて使いました、勉強になります 🙏 
https://github.com/tskaigi/tskaigi2025-web-site/pull/9/commits/9f2a69f3f39ee9ae8b59235dcb5ac91e5e92b75d